scalyr_agent/util.py

```diff
@@ -196,11 +196,14 @@ def create_unique_id():
     return result
 
 
-def md5_digest(data):
+def md5_digest(data, hexdigest=True):
```

Comment:
I'm sure it was just an oversight that we weren't using the hexdigest.  Yes, I think you can just make this always return the hex digest.